src/databricks/labs/ucx/queries/assessment/estimates/03_0_data_migration.md

```diff
@@ -12,17 +16,5 @@ Assumptions for a single table migration estimates:
 - Target Catalogs and schemas has been defined
 - Grants has been defined
 
-Please note that depending on the table type and their location, the migration effort will differ.
```

Comment:
Note that LakeView does not support markdown tables, thus converted this to a sql query, see `04_1_data_migration_efforts.sql`